spec/deserializer/acceptance_artifact_spec.rb

```diff
@@ -0,0 +1,57 @@
+# Usage: bundle exec rspec spec/deserializer/acceptance_artifact_spec.rb
+
+require 'ledger_sync'
+require 'rspec'
+
+class Package < LedgerSync::Resource
+  attribute :blabla, type: LedgerSync::Type::String
+end
+
+class Package
+  class Deserializer < LedgerSync::Deserializer
+    attribute :blabla, hash_attribute: :blabla
+  end
+end
+
+class AcceptanceArtifact < LedgerSync::Resource
+  attribute :identifier, type: LedgerSync::Type::String
+  references_many :packages, to: Package
```

Comment:
What if we add a param called `default` to `references_many` so that you could do `default: []`? I worry that your implementation here is going to be a breaking change. Unless I am missing something, I believe one could argue either way that an API not returning a parameter should result in an empty list or `nil`. So I think we should make it configurable to make this change backwards compatible.

@jozefvaclavik do you agree?

Reply:
It's exactly what I searched at the beginning, a `default`. It was the easy solution but it doesn't exist yet. So yes, it could've been a solution.

For my first implementation, yes, it was a naive approach, I didn't take time to check the whole code base. My goal was to even avoid `default` param and just do a presence check.

But still, need to think about the implementation of a `default` while not creating a breaking change.. :thinking: